more/free-programming-interactive-tutorials-en.md

```diff
@@ -223,6 +224,11 @@
 * [React Tutorial](https://react-tutorial.app)
 
 
+#### Svelte
+
+* [Svelte Tutorial](https://svelte.dev/tutorial/basics) - Svelte.dev
```

Comment:
Not interactive. It has TOC. Move it to right section in books listing.

Reply:
Moved, thank you.